### ISSUE TITLE
Missing error for `unknown var` in Mixin override and var definitions

### ISSUE THREAD

**Issue by hepiyellow** [wix/stylable#391]
In this example X variable is not defined,
but we don't get an error for it.
```
.root {
  -st-mixin: MyMixin(
    mixinArgName      value(X),
  );
}
```

**Comment by tomrav** [comment]
Indeed a bug, affecting `value` usage in Mixins with overrides and variable declarations.

**Comment by hepiyellow** [comment]
Any estimation as to when this bug would be fixed?

**Comment by hepiyellow** [comment]
@tomrav @idoros 
Hi,
I still have a hanging PR waiting for this fix.
I mean , without this fix, I can not be sure that my PR is not breaking. I need this validation.
Any estimation on when it will be fixed?

**Comment by tomrav** [comment]
It's already fixed in PR #407, hope to merge and release it tomorrow.

**Comment by tomrav** [comment]
A new version of `@stylable/core` (`v0.1.9`) was released with the requested warning, as well as a new version of `stylable-intelligence` (the extension).
